demos/projects/NXP/mimxrt1060/CMakeLists.txt

```diff
@@ -102,6 +105,7 @@ include(driver_dcp)
 include(driver_snvs_hp)
 
 add_executable(${PROJECT_NAME} ${PROJECT_SOURCES})
+add_linker(${PROJECT_NAME})
```

Comment:
what does this do?

Reply:
Adds the linker file for the sample - the ADU and non-ADU samples need different linker files now - any wording suggestions to make this clearer?